buildozer/targets/android.py

```diff
@@ -22,6 +25,7 @@
 import io
 import re
 import ast
+import sh
```

Comment:
then we may want to add the `sh` dependency to the setup.py requirement

Reply:
indeed!